charts/lakefs/values.yaml

```diff
@@ -150,3 +152,6 @@ enterprise:
         # certificates for AD FS IDP (e.g Azure AD)
         samlRsaPublicCert: null
         samlRsaPrivateKey: null
+
+secrets:
+  licenseContents: null
```

Comment:
good idea, prolly should have done those defaults for all possible secrets! 
(Not an action item just a comment)